packages/flutter/test/rendering/viewport_test.dart

```diff
@@ -1705,15 +1703,15 @@ void main() {
     final RenderViewport renderViewport = RenderViewport(
       crossAxisDirection: AxisDirection.right, offset: ViewportOffset.zero()
     );
-    FlutterError error;
+    FlutterError? error;
```

Comment:
this could be late

Reply:
Good point. I saw the `isNotNull` check expect below and thought there might be cases where it was null. But you are right, there is never a check where it is meant to be null. I'll fix this and the other. Thx.